kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/LibraryItem.vue

```diff
@@ -55,7 +55,7 @@
           :title="channel.name"
           :tagline="channel.tagline || channel.description"
           :thumbnail="channel.thumbnail"
-          :link="{}"
+          :link="genContentLinkBackLinkCurrentPage(channel.id)"
```

Comment:
These should _nearly always_ be identical, but it is safer to use `channel.root` here which is the id for the root topic, rather than the id for the channel, as `genContentLinkBackLinkCurrentPage` (and its sibling function) take a topic or resource node id.

Reply:
Thanks Richard! This has been updated